proposals/p2006.md

```diff
@@ -597,32 +665,36 @@ effective solutions.
 
 ### Alternative syntaxes for locals
 
-> TODO: needs to be rewritten
-
-**Syntactic alternatives not currently proposed:**
-
--   It is appealing to consider the `val` introducer instead of `let` given that
-    these are expected to be called "values". However, there is some concern
-    over the visual similarity of `val` and `var`, and if pronounced as written
-    instead of as "value" ar "variable", the difference relies on a subtle
-    consonant distinction not present in some widely used spoken languages. That
-    said, it is worth investigating whether there is a real user preference
-    between these introducers. Kotlin even has both `val` and `var` with a
-    similar distinction and so might serve as the basis for investigating this.
-    For now, using the less controversial spelling is proposed as this is an
-    easy thing to change later.
--   Another potential introducer would be `const`. However the semantics would
-    be subtly different even if the use cases would be similar, which might make
-    reuse of the keyword more confusing than helpful. Also, it is longer and
-    expected to be extremely common.
--   Rust uses the keyword `mut` for a similar distinction, and does not make
-    `let` optional. Carbon could adopt either aspect of those -- the keyword or
-    the requirement for `let` in every case. However, requiring two introducers
-    seems to force unnecessary ceremony into the language. The introducer `mut`
-    seems significantly less obvious to programmers familiar with any other
-    language, and isn't as obvious of an abbreviation as `var` is for
-    "variable". These issues are less pronounced when always following `let`,
-    but still present, especially nested within a pattern.
+Several other options for declaring locals were considered, but none ended up
+outweighing the proposed option on balance.
+
+-   `let` and `let mut`, based on the Rust names
+    -   While nicely non-inventive, there was both concern that `mut` didn't as
+        effectively communicate the requirement of storage and concern around
+        not being as obviously or unambiguously a good default with `let`.
+    -   The `mut` keyword feels more fitting in its use with mutable borrows
+        than as a declaration introducer the way it would work in Carbon. For
+        example, it doesn't convey a desired aspect here of allocating storage.
```

Comment:
This seems to be saying "doesn't convey allocating storage" twice, and not saying we wanted the two cases to be equally easy to say.